README.md

```diff
@@ -44,6 +44,19 @@ without having to build it from source.
 TODO(https://github.com/mc-imperial/dredd/issues/28): add instructions
 on how to use Dredd
 
+`pi.cc` is a small example in `examples/simple` that calculates an approximation of pi. To run this example, run the following from the root of the repository:
+```
+# This will save the example file since pi.cc is modified in-place by dredd.
+cp examples/simple/pi.cc examples/simple/backup.cc
+third_party/clang+llvm-13.0.1/bin/dredd examples/simple/pi.cc
+clang examples/simple/pi.cc -lstdc++ -o examples/simple/pi
```

Comment:
Change `clang` to `clang++` and then you should be able to remove `-lstdc++`. You could also note in a comment that `clang++` can be replaced with your favourite C++ compiler.

It would also be good to note that the compile command may lead to warnings being issued, due to the changes Dredd has made, and that this is OK.